wallet/models.go

```diff
@@ -579,27 +579,29 @@ type IncomingTransfersRequest struct {
 	// Return transfers for this account. (defaults to 0)
 	AccountIndex uint64 `json:"account_index,omitempty"`
 	// Return transfers sent to these subaddresses.
-	SubaddrIndices Index `json:"subaddr_indices,omitempty"`
-	// Enable verbose output, return key image if true.
-	Verbose bool `json:"verbose,omitempty"`
+	SubaddrIndices *Index `json:"subaddr_indices,omitempty"`
```

Comment:
I could be wrong, but I don't think `omitempty` works if a whole structure is not initialized, so I changed it to a pointer.